packages/@okta/i18n/src/properties/login.properties

```diff
@@ -170,6 +170,7 @@ oform.next = Next
 oform.verify = Verify
 oform.send = Send
 oform.back = Back
+oform.title = Authenticate
```

Comment:
this is too generic. maybe `oform.title.authenticate` ?

Reply:
Fixed